# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/exrtools/hotfix-zlib.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- exrtools-0.4/src/pngoutput.c.vanilla	2025-02-12 18:29:49.486564238 +0100
+++ exrtools-0.4/src/pngoutput.c	2025-02-12 18:29:56.106534111 +0100
@@ -27,6 +27,7 @@
 #include <errno.h>
 #include <string.h>
 #include <png.h>
+#include <zlib.h>
 #include "pngoutput.h"
 
 struct pngoutput_s
